setup.py

```diff
@@ -126,6 +126,9 @@
         'invenio_base.apps': [
             'invenio_oauthclient = invenio_oauthclient:InvenioOAuthClient',
         ],
+        'invenio_base.api_apps': [
```

Comment:
I am afraid about the identity expansion when used for REST requests since we need to "change identity" for each and every request.